releasenotes/notes/remove-deprecated-dbapi-config-b25bc1b236672fed.yaml

```diff
@@ -0,0 +1,5 @@
+---
+deprecations:
+  - |
+    The deprecated dbapi2 configuration has been removed. The
+    integration-specific configuration should be used instead.
```

Comment:
We should probably have an "upgrade" note in here as well giving an example of "if you were doing this... not do this"

Reply:
These were given at the time the feature was deprecated. Should I add them again here as well for the reader's convenience?

Reply:
hmm, good point.... yeah. I'd either put them here as well, or just add a note saying "Look at v0.48.0 release notes for upgrade instructions".

Reply:
I prefer adding a reference which also makes it clear when the feature was deprecated